.github/workflows/e2e-ci.yml

```diff
@@ -62,6 +62,8 @@ jobs:
         run: |
           docker build -f package/Dockerfile -t rancher/fleet:dev --build-arg="ARCH=$GOARCH" .
           docker build -f package/Dockerfile.agent -t rancher/fleet-agent:dev --build-arg="ARCH=$GOARCH" .
+          cd e2e/assets/gitrepo
+          docker build -f Dockerfile.gitserver -t nginx-git:test --build-arg="passwd=$(openssl passwd foo)" .
```

Comment:
How long does this take? Should we publish the image in a separate pipeline, so that we don't have to rebuild every time?